src/database/database.rs

```diff
@@ -20,6 +20,30 @@ impl Database {
         let conn = expect_log!(res, "Couldn't lock mutex");
         conn.execute(sql, pars)
     }
+    #[allow(dead_code)]
+    pub fn execute_batch(&self, sql: &str) -> Result<()> {
+        let res = self.conn.lock();
+        let conn = expect_log!(res, "Couldn't lock mutex");
+        conn.execute_batch(sql)
+    }
+
+    pub fn core_pragma_query(&self, pragma_variable_name: &str) -> i32 {
+        let res = self.conn.lock();
+        let conn = expect_log!(res, "Couldn't lock mutex");
+        let mut value = 0;
```

Comment:
This construct seems weird. Can't we have `pragma_query` returning the value in the result and use that?

Reply:
Result<T, Error> ?
Will check.